.github/workflows/rust.yml

```diff
@@ -12,9 +19,15 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Install additional dependencies
+        run: |
+          sudo apt install -y ${{ inputs.extra_build_dependencies }}
+        if: inputs.extra_build_dependencies != ''
```

Comment:
I would personally put the if before the run, but just style.. ^^

Reply:
I just realized that I've been putting the `if` first in the other workflows in this repo already, so I'll change it for consistency's sake